gradle/libs.versions.toml

```diff
@@ -35,13 +36,33 @@ edc-spi-catalog = { module = "org.eclipse.edc:catalog-spi", version.ref = "edc"
 edc-spi-core = { module = "org.eclipse.edc:core-spi", version.ref = "edc" }
 edc-spi-ids = { module = "org.eclipse.edc:ids-spi", version.ref = "edc" }
 edc-spi-web = { module = "org.eclipse.edc:web-spi", version.ref = "edc" }
+edc-spi-jsonld = { module = "org.eclipse.edc:json-ld-spi", version.ref = "edc" }
+edc-jsonld = { module = "org.eclipse.edc:json-ld", version.ref = "edc" }
 edc-util = { module = "org.eclipse.edc:util", version.ref = "edc" }
 failsafe-core = { module = "dev.failsafe:failsafe", version.ref = "failsafe" }
 restAssured = { module = "io.rest-assured:rest-assured", version.ref = "restAssured" }
 
+# protocol modules
+edc-dsp-api-config = { module = "org.eclipse.edc:dsp-api-configuration", version.ref = "edc" }
+edc-dsp-catalog-api = { module = "org.eclipse.edc:dsp-catalog-api", version.ref = "edc" }
+edc-dsp-catalog-http-dispatcher = { module = "org.eclipse.edc:dsp-catalog-http-dispatcher", version.ref = "edc" }
+edc-dsp-catalog-transform = { module = "org.eclipse.edc:dsp-catalog-transform", version.ref = "edc" }
+edc-dsp-http-core = { module = "org.eclipse.edc:dsp-http-core", version.ref = "edc" }
+edc-dsp-http-spi = { module = "org.eclipse.edc:dsp-http-spi", version.ref = "edc" }
+edc-dsp-negotiation-api = { module = "org.eclipse.edc:dsp-negotiation-api", version.ref = "edc" }
+edc-dsp-negotiation-http-dispatcher = { module = "org.eclipse.edc:dsp-negotiation-http-dispatcher", version.ref = "edc" }
+edc-dsp-negotiation-transform = { module = "org.eclipse.edc:dsp-negotiation-transform", version.ref = "edc" }
+edc-dsp-transfer-process-api = { module = "org.eclipse.edc:dsp-transfer-process-api", version.ref = "edc" }
+edc-dsp-transfer-process-http-dispatcher = { module = "org.eclipse.edc:dsp-transfer-process-http-dispatcher", version.ref = "edc" }
+edc-dsp-transfer-process-transform = { module = "org.eclipse.edc:dsp-transfer-process-transform", version.ref = "edc" }
+edc-dsp-transform = { module = "org.eclipse.edc:dsp-transform", version.ref = "edc" }
+
 [bundles]
-edc-connector = [ "edc-boot", "edc-core-connector", "edc-core-jersey", "edc-api-observability" ]
-edc-dpf = [ "edc-dpf-transferclient", "edc-dpf-selector-client", "edc-dpf-selector-spi", "edc-dpf-selector-core", "edc-dpf-framework" ]
+edc-connector = ["edc-boot", "edc-core-connector", "edc-core-jersey", "edc-api-observability"]
+edc-dpf = ["edc-dpf-transferclient", "edc-dpf-selector-client", "edc-dpf-selector-spi", "edc-dpf-selector-core", "edc-dpf-framework"]
+edc-dsp = ["edc-dsp-api-config", "edc-dsp-catalog-api", "edc-dsp-catalog-http-dispatcher", "edc-dsp-catalog-transform", "edc-dsp-http-core", "edc-dsp-http-spi",
```

Comment:
this is already defined in edc, is the `dsp` BOM module, it would be enough to import that

Reply:
there was a small but pesky bug, which prevented me from import the root bom, but I fixed it in https://github.com/eclipse-edc/Connector/pull/3013. can remove those modules